docs/guide/debugging.md

```diff
@@ -45,3 +45,17 @@ JavaScript file | ./node_modules/vitest/vitest.mjs
 Application parameters | run --threads false
 
 Then run this configuration in debug mode. The IDE will stop at JS/TS breakpoints set in the editor.
+
+## Node Inspector, e.g. Chrome DevTools
+
+Vitest also supports debugging tests without IDEs. However this requires that tests are not run parallel. Use one of the following commands to launch Vitest.
```

Comment:
Should Vitest fail with a nice error message, if it's not running with either of these? Validation can be done in `node/config.ts`

Reply:
Added validation for this. Looks like `node:test` has similar behaviour: https://github.com/nodejs/node/blob/c733cc0c7f34916dde4808716e32df62f335dc30/lib/internal/main/test_runner.js#L19

I also noticed that the errors thrown in `node/config.ts` are not shown to users. I'll fix that next and add some test cases.

```sh
$ pnpm run test --shard 2 --watch

> example-project@1.0.0 test /x/vitest-example-project
> vitest run "--shard" "2" "--watch"

 ELIFECYCLE  Test failed. See above for more details.
```